technical-specs/index.md

```diff
@@ -3,4 +3,133 @@ layout: custom
 title: titles.technicalspecs
 permalink: /technical-specs/index.html
 ---
-{% tf technical-specs.md %}
\ No newline at end of file
+<div class="about-monero">
+    <section class="container">
+        <div class="row">
+            <!-- left two-thirds block-->
+            <div class="full col-xs-12">
+                <div class="info-block text-adapt">
+                    <div class="row">
+                        <div class="col">
+                            <h3>No premine, no instamine, no token</h3>
+                        </div>
+                    </div>
+                    <div>
+                        <ul>
+                            <li>Monero had no premine or instamine</li>
+                            <li>Monero did not sell any token</li>
+                            <li>Monero had no presale of any kind</li>
+                        </ul>
```

Comment:
This needs to be done too. I can have a look at it if you want @rehrar 
I can see that the _i18n still contains the tech-specs file, so i believe you just forget about it.